.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -10,9 +10,14 @@ trigger:
   paths:
     exclude:
       - docs/*
-      - tracer/README.md
+      - tracer/README.MD
       - .github/*
       - tracer/dependabot/*
+      - tracer/src/Datadog.Trace/DuckTyping/README.md
+      - blog/*
```

Comment:
Wildcards!
```suggestion
      - .github/
      - tracer/dependabot/
      - tracer/src/Datadog.Trace/DuckTyping/README.md
      - blog/
```

Reply:
so looking at the doc wildcards are supported in path but only for the push trigger: 
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#wildcards
that's why I left them

Reply:
Yes I agree. As those were already there originally, I didn't want to do any more changes than necessary TBH. 2 failed attempts are already too much

Reply:
> so looking at the doc wildcards are supported in path but only for the push trigger

What were they thinking 🤦‍♂️